packages/terra-form/src/NumberField.jsx

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Field from './Field';
+import Input from './Input';
+
+const propTypes = {
+  /**
+   * The defaultValue of the input element
```

Comment:
If I remember correctly this prop indicates the difference between a controlled & uncontrolled component?